contentcuration/contentcuration/static/js/edit_channel/uploader/views.js

```diff
@@ -346,12 +339,31 @@ var EditMetadataList = BaseViews.BaseEditableListView.extend({
       this.shared_data.shared_author = view.model.get("author");
       this.shared_data.shared_license = view.model.get("license");
       this.shared_data.all_files = view.model.get("kind") !== "topic";
+      this.shared_data.all_exercises = view.model.get("kind") === "exercise";
```

Comment:
see if all selected items are exercises to render mastery model options correctly